_data/simple-icons.json

```diff
@@ -1770,6 +1770,11 @@
             "hex": "4B4B77",
             "source": "https://github.com/avajs/ava/blob/6f8e30c94626238a5b26deadac319089fa43d333/media/logo.svg"
         },
+        {
+            "title": "Avalonia UI",
+            "hex": "8B44AC",
+            "source": "https://github.com/AvaloniaUI/Avalonia/blob/master/build/Assets/Icon.png"
```

Comment:
There is an SVG version at https://github.com/AvaloniaUI/avalonia-docs/blob/main/static/logo/avalonia.svg.